### PR TITLE
Fk pd 2513 reblock gvcf

### PR DESCRIPTION
**Pull request by khajoue2**
### Description
Updated ReblockGVCF.wdl to run both in Azure and gcp
Updated all test inputs for scientifc and plumbing
Updated verification wdl for testing
Added new versions to dependent pipelines.

- [ X] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ X] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by nikellepetrillo — APPROVED** [review]
looks good to me! thank you

**Review by nikellepetrillo — COMMENTED** [review]
it looks like womtools check is failing and the following need updates because a shared task (validateVCF) was updated:
ExternalExomeReprocessing.wdl:
ExternalWholeGenomeReprocessing
ExomeReprocessing.wdl
WholeGenomeReprocessing.wdl

File: pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.1.20
+2024-04-08 (Date of Last Commit)
+
+* Updated tasks GermlineVariantDiscovery.wdl and QC.wdl to allow multi-cloud dockers; this does not affect this pipeline.
```
Comment (ekiernan) [inline pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md:4]: think we need to remove period at the ends

**Review by kevinpalis — APPROVED** [review]

File: pipelines/broad/dna_seq/germline/single_sample/exome/test_inputs/Scientific/D5327.NA12878.json [hunk 2]
```diff
@@ -56,5 +56,6 @@
   },
 
   "ExomeGermlineSingleSample.AggregatedBamQC.CollectReadgroupBamQualityMetrics.collect_gc_bias_metrics": false,
-  "ExomeGermlineSingleSample.AggregatedBamQC.CollectAggregationMetrics.collect_gc_bias_metrics": false
+  "ExomeGermlineSingleSample.AggregatedBamQC.CollectAggregationMetrics.collect_gc_bias_metrics": false,
+  "ExomeGermlineSingleSample.cloud_provider": "gcp"
```
Comment (ekiernan) [inline pipelines/broad/dna_seq/germline/single_sample/exome/test_inputs/Scientific/D5327.NA12878.json:60]: is there an extra space at the beginning of this line?

File: pipelines/broad/dna_seq/germline/single_sample/exome/test_inputs/Scientific/D5327.NA12891.json [hunk 3]
```diff
@@ -56,5 +56,6 @@
   },
 
   "ExomeGermlineSingleSample.AggregatedBamQC.CollectReadgroupBamQualityMetrics.collect_gc_bias_metrics": false,
-  "ExomeGermlineSingleSample.AggregatedBamQC.CollectAggregationMetrics.collect_gc_bias_metrics": false
+  "ExomeGermlineSingleSample.AggregatedBamQC.CollectAggregationMetrics.collect_gc_bias_metrics": false,
+  "ExomeGermlineSingleSample.cloud_provider": "gcp"
```
Comment (ekiernan) [inline pipelines/broad/dna_seq/germline/single_sample/exome/test_inputs/Scientific/D5327.NA12891.json:60]: spacing line 60

File: pipelines/broad/dna_seq/germline/single_sample/ugwgs/UltimaGenomicsWholeGenomeGermline.changelog.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 1.0.17
+2024-04-08 (Date of Last Commit)
+
+* Changed ReblockGVCFs.wdl to be multicloud.
```
Comment (ekiernan) [inline pipelines/broad/dna_seq/germline/single_sample/ugwgs/UltimaGenomicsWholeGenomeGermline.changelog.md:4]: need to remove period at the end

File: pipelines/broad/dna_seq/germline/single_sample/wgs/test_inputs/Plumbing/dragen_mode_best_results.json [hunk 5]
```diff
@@ -62,5 +62,6 @@
   },
 
   "WholeGenomeGermlineSingleSample.dragen_maximum_quality_mode": true,
-  "WholeGenomeGermlineSingleSample.UnmappedBamToAlignedBam.CheckContamination.disable_sanity_check": true
+  "WholeGenomeGermlineSingleSample.UnmappedBamToAlignedBam.CheckContamination.disable_sanity_check": true,
+  "WholeGenomeGermlineSingleSample.cloud_provider": "gcp"
```
Comment (ekiernan) [inline pipelines/broad/dna_seq/germline/single_sample/wgs/test_inputs/Plumbing/dragen_mode_best_results.json:66]: spacing different on these jsons too

**Review by ekiernan — APPROVED** [review]
Just a few minor formatting comments, but LGTM! A few jsons had what looked like some extra spaces at the beginning of the line and the changelogs had some periods at the end of the bullet.

**Pull request merged**